### DISCUSSION TITLE
Designing a host PCB to simplify wiring and putting into an enclosure.

### DISCUSSION THREAD

**Discussion by multigamesystem** [OrangeFox86/DreamPicoPort#118] [General]
Ive been designing a 3D printable case for the the Pico + Dreamcast extension cable end.  So far its gone pretty well with just wiring directly to the pico but after a few attempts Ive realized its easy to damage wires and the resistors are a pain point.  So Ive designed a PCB to simplify the wiring.  I was gonna design around the recommended 74LVC2T45DC logic package but I ran out of steam.  Whoops forgot to change the text from 1 Player to 4 player.  I just modified the board to have 3 more player inputs so I didnt waste time making 2 different PCBs.  Oh well no big deal.

![DC Pink Pico to USB C v21](https://github.com/user-attachments/assets/3b48912b-7148-4de0-8332-26f05ac3e022)

![IMG_20240301_133216_448](https://github.com/user-attachments/assets/09575b29-40a8-4c63-aa64-2b624b46addc)

**Comment by multigamesystem** [comment]
Crap @Tails86 I accidentally wired up player 2 to GP16 + GP17 I sent off to get the boards made already.  How hard is to to modify the program?  Dont judge my schematic Im not an electrical engineer by any means.  
![Screenshot 2024-07-14 012152](https://github.com/user-attachments/assets/6c442e47-a62b-4f61-9838-f262de1b669e)

**Reply by Tails86** [reply]
Not hard. Definitions are located here: https://github.com/OrangeFox86/DreamcastControllerUsbPico/blob/main/inc/configuration.h#L48

**Reply by multigamesystem** [reply]
I found those but how would I go about modifying them and recompiling into a pico program?

**Reply by Tails86** [reply]
Build instructions are on the readme: https://github.com/OrangeFox86/DreamcastControllerUsbPico?tab=readme-ov-file#build-instructions-for-linux-and-windows
I created this branch and included the uf2 files: https://github.com/OrangeFox86/DreamcastControllerUsbPico/tree/p2-pin-16/dist
(change was not tested)

**Reply by multigamesystem** [reply]
Thanks for your help @Tails86 !  Ill test and report back when the pcbs arrive.  Should be here by Monday.

**Reply by browningonline** [reply]
This looks great! I tackled a similar problem but had a board made using HDMI ports for all 4 players.  I swapped 3 and 4 though and need to change the firmware.  I am using HDMI since the adapters I have are from the Bliss-Box 4Play.  Where can I source the extension cables you have with the 4 pin sockets?
